shared/js/background/safari-wrapper.es6.js

```diff
@@ -1,3 +1,6 @@
+/* global safari:false */
+const tabManager = require('./tab-manager.es6')
+
```

Comment:
cc @jdorweiler - `tabManager` is used elsewhere in the file, but it wasn't defined. Could that be behind all the weirdness you were seeing?

Reply:
Defined as a global here for the UI to use: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/master/shared/js/background/safari-events.es6.js#L10

Doesn't hurt to have it required here just in case we ever fix the UI active tab problem.